UIKitLearnings/UIKitLearnings/Animations/AnimationPlayViewController.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  AnimationPlayViewController.swift
+//  UIKitLearnings
+//
+//  Created by Saurabh Verma on 04/06/24.
+//
+
+import UIKit
+
+class AnimationPlayViewController: UIViewController {
+
+    var animationOption: AnimationsOption!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        if let animatingView = getAnimationView() {
+            self.view.addSubview(animatingView)
+        }
+    }
+    
+    func getAnimationView() -> UIView? {
+        if let animationOption = animationOption {
+            switch animationOption {
+            case .circularProgressView:
+                return setUpCircularProgressView()
+            case .imageView:
+                return setUpAnimatingImageView()
+            case .rectangularView:
+                return setUpRectangularAnimatingView()
+            }
+        }
+        return nil
+    }
+}
+
+extension AnimationPlayViewController {
+
+    func setUpCircularProgressView() -> UIView {
+        let frame = CGRect(x: 100, y: 200, width: 200, height: 200)
```

Comment:
Add a TODO, to properly set this in parent view instead of setting some random values.